setup.py

```diff
@@ -51,7 +51,7 @@ def run(self):
     author_email='pretix-eth-payment-plugin@ethereum.org',
     license='Apache Software License',
     install_requires=[
-        "Django==2.2.2",
+        "Django==2.2.*",
```

Comment:
```suggestion
        "Django>=2.2.0,<2.3.0",
```
This is a more canonical way to specify that version constraint.  In fact, I'm not sure the `*` syntax will work.  Never used it.